src/main/java/org/omnifaces/util/Utils.java

```diff
@@ -288,9 +290,10 @@ public static boolean isNumber(String string) {
     public static boolean isDecimal(String string) {
         try {
             // Performance tests taught that this approach is in general faster than regex or char-by-char checking.
-            return Double.valueOf(string) != null;
+            Double.parseDouble(string);
```

Comment:
Sonar will give a warning on this. Please restore `!= null`.

Reply:
These new methods returns a primitive value, so if we and a `... != null` wan't compile at all...